components/Hubs/MyHubsDropdown.tsx

```diff
@@ -0,0 +1,120 @@
+import DropdownButton from "~/components/Form/DropdownButton";
+import colors from "~/config/themes/colors";
+import { useEffect, useMemo } from "react";
+import { StyleSheet, css } from "aphrodite";
+import { connect } from "react-redux";
+import Link from "next/link";
+import icons from "~/config/themes/icons";
+import { breakpoints } from "~/config/themes/screen";
+
+type Args = {
+  hubState?: any;
+  isOpen?: boolean;
+};
+
+const MyHubsDropdown = ({ hubState, isOpen = false }: Args) => {
+  const renderDropdownOpt = (hub) => {
+    return (
+      <Link href={`/hubs/${hub.slug}`} key={hub.id}>
+        <a className={css(styles.hubOpt)}>
+          <img
+            className={css(styles.hubImage)}
+            src={
+              hub.hub_image
+                ? hub.hub_image
+                : "/static/background/hub-placeholder.svg"
+            }
+            alt={hub.name}
+          />
+          <div className={css(styles.hubDetails)}>
+            <div className={css(styles.hubName)}>{hub.name}</div>
+          </div>
+        </a>
+      </Link>
+    );
+  };
+
+  const hubOpts = useMemo(() => {
+    const myHubsOpts = (hubState.subscribedHubs || []).map((h) =>
+      renderDropdownOpt(h)
+    );
+    return myHubsOpts;
```

Comment:
Explicit nullcheck. "Or" operator is based on boolean. 0 / null / empty strings / undefined are all falsey. Better to be type-specific. Prevents bugs this way.
```suggestion
    return (hubState.subscribedHubs ?? []).map((h) =>
      renderDropdownOpt(h)
    );
```

Reply:
Thanks for the suggestion.
I try to avoid using `??` unless I am working with values that require it. For example, if I were working with an integer that evaluated to 0.  I prefer continued use of the `||` operator as it makes me think of types more carefully and makes the code a bit easier to read since the double pipe exists in most languages.

LMK if you think my statement can fail due to a value I am not taking into account.

Reply:
For this speicific example, it works. 
But, there's good use case for being specific with null / falsey values, especially when it comes to upserting data.
Not the ideal situation, but as a programmer, where null & false values make huge differences. 
Let's take RSC as an example: 
Account value of `undefined` or `null` maybe

Reply:
You make a good point. I would actually want the application to break if the value were non-array.
So 0.map would cause the app the break and us to question what's going on. I like it.